### PR TITLE
 emacs: update to 24.5

### PR DESCRIPTION
**Pull request by avsej**

### CODE REVIEW

**Comment by AndWac** [comment]
File Changes

```
A components/emacs/.gitignore (1)
```

Are you sure about adding .gitignore?

**Comment by avsej** [comment]
@AndWac I did it previously it helps to keep tree clean and local to this component only

File: components/emacs/Makefile [hunk 1]
```diff
@@ -23,18 +23,18 @@
 include ../../make-rules/shared-macros.mk
```
Comment (pyhalov) [inline components/emacs/Makefile]: It would be good also to move emacs to appropriate category

Comment (avsej) [inline components/emacs/Makefile]: @pyhalov like what?

Comment (avsej) [inline components/emacs/Makefile]: vim and nano on the same level

Comment (pyhalov) [inline components/emacs/Makefile]: Either use text/ or create editor/ for them. Don't know.

Comment (avsej) [inline components/emacs/Makefile]: moved to editor/

**Pull request merged**